plugin/src/main/java/com/denizenscript/denizen/events/player/PlayerCraftsItemScriptEvent.java

```diff
@@ -58,13 +61,10 @@ public PlayerCraftsItemScriptEvent() {
 
     @Override
     public boolean couldMatch(ScriptPath path) {
-        if (!path.eventArgsLowEqualStartingAt(0, "player", "crafts")) {
+        if (!path.eventLower.startsWith("player crafts ")) {
             return false;
         }
-        if (!couldMatchItem(path.eventArgLowerAt(2))) {
-            return false;
-        }
-        return true;
+        return couldMatchItem(path.eventArgLowerAt(2));

```

Comment:
That syntax was intentional - either way if anything just update it to `registerCouldMatcher`, no point changing around code that needs to be removed anyway